en/integrations/plugin-packs/procedures/applications-monitoring-centreon-ha.md

```diff
@@ -5,7 +5,7 @@ title: Centreon-HA
 
 ## Overview
 
-Centreon-HA is Centreon central server's high availability implementation. More information available [here](administration/centreon-ha/architecture.md).
+Centreon-HA is Centreon central server's high availability implementation. More information available [here](administration/centreon-ha/overview.md).
```

Comment:
shouldn't it be this `installation/installation-of-centreon-ha/overview.md` instead of `administration/centreon-ha/overview.md` here ?